database/migrations/2020_04_30_215425_add_discord_id_column.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDiscordIdColumn extends Migration
+{
+    /**
+     * Run the migrations.t.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('mship_account', function (Blueprint $table) {
+            $table->string('discord_id', 30)->unique()->after('slack_id')->nullable();
```

Comment:
Do we also want a migration to remove slack_id's?

Reply:
Not for the time being, we'll remove it later